ansible/roles/dev_tools/tasks/main.yml

```diff
@@ -1,20 +1,15 @@
-# This role could be made more idempotent by requesting that packages be
-# "present," not "latest." The vim packaging issue could be solved with logic
-# that says "if the current platform is Fedora and vim-enhanced isn't installed,
-# let vim-minimal be latest."
-
 # Fedora has a packaging problem where vim-enhanced will fail to install if
-# vim-minimal is out of date.
+# vim-minimal is out of date. The vim packaging issue could be solved with logic
+# that says "if the current platform is Fedora and vim-enhanced isn't installed,
+# install vim-minimal."
 - block:
 
-  - name: Install latest version of vim-minimal
+  - name: Install vim-minimal
     package:
       name: vim-minimal
-      state: latest
-    tags:
-      - skip_ansible_lint
+      state: present
```

Comment:
failing for me (see my comment) :-/

Reply:
Adapted from an IRC conversation:

> I can't actually tell you why vim-minimal needs to be installed first. There was already code that did that when I first inherited this code base. But it didn't link to a bug tracker issue, or some other detailed explanation of why the strange installation method was used. If we're installing developer-friendly tools, the full vim seems like a good choice, not vim-minimal.

Reply:
I've just tried without the`vim-minimal` and with the regular `vim` instead of the `vim-enhanced` package and it has [this conflict](https://paste.fedoraproject.org/paste/VizxnTygNQ0deTlxGA7l7w). On the other hand, explicitly dropping `vim-minimal`here [conflicts with sudo](https://paste.fedoraproject.org/paste/6fW-Ht1Sg091ME9ZeKOFuw).

Seems this may need some workaround